--- a/drivers/extcon/extcon-ktm5030.c
+++ b/drivers/extcon/extcon-ktm5030.c
@@ -402,15 +402,21 @@
 			struct ktm5030 *pdata, bool show)
 {
 	bool ret = true;
-	u8 databuf[DATA_BUF_MAX_SIZE];
+	u8 *databuf;
 	const u8 *fdata = fw->data;
 	int dlen = (int)fw->size;
 	int start_addr = 0, wlen = 0, *ptr, crc16;
 	int total_page = 0, rest_data = 0, i = 0;
 
+	databuf = kmalloc(DATA_BUF_MAX_SIZE, GFP_KERNEL);
+	if (!databuf)
+		return false;
+
 	if ((pdata->fw_status != UPDATE_DRIVERWRITE) &&
-		(pdata->fw_status != UPDATE_ERASEBANK))
+		(pdata->fw_status != UPDATE_ERASEBANK)) {
+		kfree(databuf);
 		return false;
+	}
 
 	total_page = dlen / DATA_PAGE_SIZE;
 	rest_data = dlen % DATA_PAGE_SIZE;
@@ -498,6 +504,7 @@
 			pdata->fw_status = UPDATE_COREIMG;
 	}
 
+	kfree(databuf);
 	return ret;
 }
 
