--- a/include/linux/seccomp.h
+++ b/include/linux/seccomp.h
@@ -36,7 +36,6 @@
 struct seccomp {
 	int mode;
 	atomic_t filter_count;
-	atomic_t filter_count;
 	struct seccomp_filter *filter;
 };
 
--- a/security/selinux/include/objsec.h
+++ b/security/selinux/include/objsec.h
@@ -193,10 +193,4 @@
 {
 	return superblock->s_security + selinux_blob_sizes.lbs_superblock;
 }
-
-static inline struct inode_security_struct *selinux_inode(
-						const struct inode *inode)
-{
-	return inode->i_security;
-}
 #endif /* _SELINUX_OBJSEC_H_ */
